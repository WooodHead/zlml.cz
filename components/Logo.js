// @flow

import Link from 'next/link';

import Colors from '../services/Colors';

export default () => (
  <h1>
    <Link prefetch href="/">
      <a>Martin Zl√°mal</a>
    </Link>
    <style jsx>{`
      h1 a,
      h1 a:hover,
      h1 a:focus,
      h1 a:visited {
        color: ${Colors.red};
        text-decoration: none;
        font-size: 10rem;
      }
    `}</style>
  </h1>
);
