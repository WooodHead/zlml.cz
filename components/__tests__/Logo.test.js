// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Logo from '../Logo';

it('renders correctly', () => {
  expect(renderer.create(<Logo />).toJSON()).toMatchSnapshot();
});
