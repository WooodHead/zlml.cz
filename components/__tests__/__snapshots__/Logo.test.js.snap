// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <p
    style={
        Object {
            "fontSize": "2rem",
            "hyphens": "auto",
            "textAlign": "justify",
          }
    }
>
    ¡Hola! My name is
</p>,
  <h1
    className="jsx-2345896288"
>
    <span
        className="jsx-364771521"
    >
        <a
            className="jsx-364771521"
            href="/"
            onClick={[Function]}
        >
            Martin Zlámal
        </a>
    </span>
</h1>,
]
`;
